jest.config.js

```diff
@@ -1,14 +1,16 @@
 module.exports = {
+    bail: true,
     verbose: true,
     collectCoverage: true,
-    collectCoverageFrom: ["**/*.{tsx}", "!**/node_modules/**", "!**/dist/**"],
-    setupFiles: ["<rootDir>/ui-tests/jestsetup.ts"],
+    coverageDirectory: "<rootDir>/coverage/",
+    collectCoverageFrom: ["**/jest-tests/*.{tsx}", "!**/node_modules/**", "!**/dist/**"],
```

Comment:
Does this enable code coverage for our `tsx` files now?

Reply:
On going experimentation I though on reading the docs that just adding that might be enough basically `jest` works with istanbul built in and can output coverage reports even in ts straight forwardly apparently but not sure if they are being picked up or im missing a step somewhere.